src/ripe/order.py

```diff
@@ -324,11 +324,21 @@ def update_tag(self, number, identifier=None, type=None, activate=None):
         contents = self.put(url, identifier=identifier, type=type, activate=activate)
         return contents
 
+    def delete_tag(self, number, deactivate=None):

```

Comment:
Ok, I notice in the function above I used None for default as well, but should'nt it be set to True ? By the logic of lines like: https://github.com/ripe-tech/ripe-api/pull/44/files#diff-6c079188dcb0aa30585736d9920b2bb43125a475785ada56fc98678e6611776cR310